frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -218,48 +223,70 @@ const TranslationPage = () => {
       </Box>
     ) : null;
   };
-
   return (
-    <Box margin="20px 20px 0px 20px">
-      <Text size="lg">Story Title Here</Text>
-      <Text as="ins">View story details</Text>
-      <FontSizeSlider setFontSize={handleFontSizeChange} />
-      <Flex>
-        <Flex direction="column" width="75vw">
-          <Flex direction="row">
-            <IconButton
-              variant="ghost"
-              aria-label="Undo Change"
-              onClick={undoChange}
-              icon={<Icon as={MdUndo} />}
```

Comment:
oh shoot shouldve mentioned - designers were cool with using the `<Icon as={MdUndo} />` from "@chakra-ui/react". mind removing the svgs and switching back to these?